config/application.yml.example

```diff
@@ -50,7 +48,7 @@ development:
   RAILS_ENV:                 'development'
   RACK_ENV:                  'development'
   DB_NAME:                   'barito_development'
-  ENABLE_CAS_INTEGRATION:    'false'
+  ENABLE_SSO_INTEGRATION:    'false'
 
```

Comment:
Please add `whitelisted_email_domains` here.

Reply:
> Please also add `whitelisted_email_domains, GOOGLE_CLIENT_ID, GOOGLE_CLIENT_SECRET` sample here.

Made a pull request (https://github.com/BaritoLog/BaritoMarket/pull/253) for the `google-sso` branch to fulfill this.